lite/example2/spdx-3.0/lite-example-2.spdx.json

```
@@ -0,0 +1,136 @@
+                "build"
+            ]
+        },
+        {
+            "type": "CreationInfo",
+            "@id": "_:creationinfo",
+            "specVersion": "3.0.0",
+            "comment": "if any",
+            "created": "2024-05-06T00:00:00Z",
+            "createdBy": "https://spdx.org/spdxdocs/08f113e9-a0b0-4482-a0ed-c4e18e5136be/Agent/NorioKobota"
+        },
+        {
+            "type": "Person",
+            "spdxId": "https://spdx.org/spdxdocs/08f113e9-a0b0-4482-a0ed-c4e18e5136be/Agent/NorioKobota",
+            "name": "Norio Kobota",
+            "creationInfo": "_:creationinfo",
+            "externalIdentifier": {
+                "type": "ExternalIdentifier",
+                "externalIdentifierType": "email",
+                "identifier": "norio.kobota@sony.com"
+            }
+        },
+        {
+            "type": "software_Package",
+            "spdxId": "https://spdx.org/spdxdocs/08f113e9-a0b0-4482-a0ed-c4e18e5136be/Package/1",
+            "name": "my-package",
+            "comment": "if any",
+            "creationInfo": "_:creationinfo",
+            "verifiedUsing": [{
+                "type": "Hash",
+                "algorithm": "sha3_512",
+                "hashValue": "hash value of the package file"
+            }],
+            "originatedBy": [
+                "https://spdx.org/spdxdocs/08f113e9-a0b0-4482-a0ed-c4e18e5136be/Agent/NorioKobota"
+            ],
+            "suppliedBy": "https://spdx.org/spdxdocs/08f113e9-a0b0-4482-a0ed-c4e18e5136be/Agent/NorioKobota",
+            "builtTime": "2024-05-06T00:00:00Z",
+            "releaseTime": "2024-05-06T00:00:00Z",
+            "validUntilTime": "2034-05-06T00:00:00Z",
+            "supportLevel": "limitedSupport",
+            "software_copyrightText": "copyright text",
+            "software_attributionText": "other attribution text",
+            "software_packageVersion": "v1.0",
+            "software_downloadLocation": "http://dl.example.com/my-package_v1.0.tar",
+            "software_packageUrl": "pkg:github/example/my-package/releases/tag/v1.0",
+            "software_homepage": "website for the Package/1"
+        },
+        {
+            "type": "software_Package",
+            "spdxId": "https://spdx.org/spdxdocs/08f113e9-a0b0-4482-a0ed-c4e18e5136be/Package/2",
+            "name": "my-package_1.0.0.tar.gz",
+            "creationInfo": "_:creationinfo",
+            "verifiedUsing": [{
+                "type": "Hash",
+                "algorithm": "sha3_512",
+                "hashValue": "hash value of my-package_1.0.0.tar.gz file"
+            }]
+        },
+        {
+            "type": "simpleLicensing_LicenseExpression",
+            "spdxId": "https://spdx.org/spdxdocs/08f113e9-a0b0-4482-a0ed-c4e18e5136be/LicenseExpression/1",
+            "creationInfo": "_:creationinfo",
+            "simpleLicensing_licenseExpression": "GPL-2.0-only",
+            "simpleLicensing_licenseListVersion": "3.23.0"
```

```suggestion
            "simplelicensing_licenseExpression": "GPL-2.0-only",
            "simplelicensing_licenseListVersion": "3.23.0"
```

lowercase L for licensing